kustomize/commands/internal/util/util_test.go

```diff
@@ -89,8 +89,9 @@ func TestGlobPatternsWithLoaderRemoteFile(t *testing.T) {
 	}
 
 	// test load invalid file
-	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{"http://invalid"})
-	if err != nil {
+	invalidURL := "http://invalid"
+	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{invalidURL})
+	if err != nil && err.Error() != invalidURL+" has no match: "+invalidURL+" not exist" {
```

Comment:
Can we also add a check here for 

```
if err == nil {
    t.Fatalf("expected error but did not receive one")
}
```

Reply:
I fixed it at https://github.com/kubernetes-sigs/kustomize/pull/5030/commits/8383b2832290dc969f2924a1c552852159bbaeed.